test_wrapper/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.9"
```

Comment:
you don't need to worry about the docker compose here. We can do the docker compose stuff as part of the test runner

Reply:
@Andrew-Hayes I have used it for my development and testing. Is it ok to leave compose file?